pkg/resource/agents-toggle/observability_bundle_configmap.go

```diff
@@ -59,11 +59,32 @@ func GenerateObservabilityBundleConfigMap(ctx context.Context, lc loggedcluster.
 		return v1.ConfigMap{}, errors.Errorf("unsupported logging agent %q", lc.GetLoggingAgent())
 	}
 
-	if observabilityBundleVersion.GE(semver.MustParse("0.10.0")) {
```

Comment:
We still have a few 0.10.1 observability bundles. Do we not need this?

Reply:
Oh well no we do not

Reply:
I guess that [this](https://github.com/giantswarm/logging-operator/blob/add-events-logger-toggle/pkg/resource/agents-toggle/observability_bundle_configmap.go#L63) covers these cases right ?

Reply:
Well no because the Idea was to not deploy the grafana agent before v0.10.0 but we dont run those anymore anyway

Reply:
Ok added it back in a different way. All good for you ?

Reply:
Ok fine with me :)